src/map/src/cmapclient.cpp

```diff
@@ -229,7 +229,7 @@ bool CMapClient::joinServerReply(RoseCommon::Packet::CliJoinServerReq&& P) {
 
           if(position.map == 20)
           {
-            auto spawn = Combat::get_spawn_point(*entitySystem, entity);
+            auto spawn = Combat::get_spawn_point(*entitySystem, entity, 3);
```

Comment:
what's this 3 magic number?

Reply:
yes please change to `CliReviveReq::ReviveRequest::START_POSITION`

Reply:
Marking as request for changes